GAD 7 - Radixsort/tests/radix/RadixsortTester.java

```diff
@@ -0,0 +1,821 @@
+package tests.radix;
+
+import gad.radix.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class RadixsortTester {
```

Comment:
The class should be named RadixSortTester

Reply:
Renamed